msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp

```diff
@@ -367,9 +367,7 @@
 
             site-id='<h:outputText value="#{ForumTool.siteId}" />'
 			dont-associate-label='<h:outputText value="#{msgs.forum_dont_associate_label}" />'
-			dont-associate-value="0"
```

Comment:
I remember fixing bugs in Rubrics and these lines were useless, is this a cleanup?

Reply:
Yes, this is useless and I also needed a '2' value, so I removed the property-passing bits from all the instances.

Reply:
Nice